src/qml/BaseMovesComponent.qml

```diff
@@ -130,38 +133,19 @@ Item {
                 }
             }
         }
-        Item {
-            anchors {
-                top: parent.top;
-                left: parent.left;
-                right: parent.right;
-                margins: -Kirigami.Units.gridUnit;
-            }
-            height: parent.height > root.height ? (parent.height + Kirigami.Units.gridUnit*2) : (root.height - Kirigami.Units.gridUnit*2 - Kirigami.Units.largeSpacing);
-            opacity: (CommandQueue.currentCommandRemainingMSeconds > 0 && root.blockOnMovingTail === true) ? 1 : 0;
-            Behavior on opacity { PropertyAnimation { duration: Kirigami.Units.shortDuration; } }
-            MouseArea { anchors.fill: parent; enabled: parent.opacity > 0; onClicked: { /* nothing, because we're just swallowing events */ } }
-            Rectangle { anchors { fill: parent; margins: Kirigami.Units.smallSpacing; } color: "black"; opacity: 0.8; radius: Kirigami.Units.smallSpacing; }
-            Kirigami.Heading {
-                anchors {
-                    left: parent.left;
-                    right: parent.right;
-                    top: parent.top;
-                    topMargin: Kirigami.Units.gridUnit * 4;
-                }
-                color: "white";
-                text: qsTr("Tail Active...");
-                horizontalAlignment: Text.AlignHCenter;
-                ProgressBar {
-                    anchors {
-                        left: parent.left;
-                        right: parent.right;
-                        top: parent.bottom;
-                        margins: Kirigami.Units.gridUnit;
-                    }
-                    value: CommandQueue.currentCommandRemainingMSeconds;
-                    to: CommandQueue.currentCommandTotalDuration;
-                }
+    }
+
+    ActiveMovePopup {
+        id: activeMovePopup
+
+        x: (root.width - width) / 2
+        y: (root.height - height) / 2
+
+        Connections {
+            target: control
+
+            onCommandActivated: {
+                activeMovePopup.commandName = name;
```

Comment:
Hmm... i was thinking of exposing it through the command queue, but this works :) Perhaps give the signal's command name parameter a slightly less generic name than name? (it's called "command" pretty much everywhere else, so that'd probably do the trick... just thinking that i'll be coming back to this in six months and it'll make me very confused what "name" is ;) )